ansible/configs/pntae-app/files/cloud_providers/ec2_cloud_template.j2

```diff
@@ -0,0 +1,287 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Mappings:
+  RegionMapping:
+    us-east-1:
+      RHELAMI: ami-c998b6b2
```

Comment:
Use the RHEL GOLD AMI as in https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles-infra/infra-ec2-template-generate/defaults/main.yml#L100